src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -138,7 +138,6 @@ public enum ReportNameMatchingType {
         EXACT_MATCHING, ENDS_WITH
     }
 
-    // TODO remove ComputationType (and use ReportType instead) after short-circuit logs have been invalidaded with admin-tool
```

Comment:
Shouldn't we keep it as ComputationType is not completly removed ?

Reply:
Sadly we have to keep the two enums, so the TODO is useless.